### DISCUSSION TITLE
白屏了，已经不维护了吗

### DISCUSSION THREAD

**Discussion by Snorlaxx-plot** [alibaba/designable#402] [Q&A]
<img width="810" alt="image" src="https://github.com/user-attachments/assets/b9e4e411-2bfe-479c-92f2-5fafecb2fb3f" />

**Comment by haiboge** [comment]
同上。。。

**Comment by nan-184** [comment]
![image](https://github.com/user-attachments/assets/e69b3138-d5b9-4e43-a91e-14ba6a40590c)

**Reply by nan-184** [reply]
貌似好久没有更新了

**Comment by erweixin** [comment]
部署了一版，要用的话可以参考下 [designable-antd.vercel.app/](https://designable-antd.vercel.app/) @nan-184

**Reply by nan-184** [reply]
这个链接也打不开
![image](https://github.com/user-attachments/assets/44322ed7-df5d-42de-8825-8e6502a51511)

**Reply by erweixin** [reply]
> 这个链接也打不开 ![image](https://private-user-images.githubusercontent.com/73976796/405516338-44322ed7-df5d-42de-8825-8e6502a51511.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc1MzI0MTEsIm5iZiI6MTczNzUzMjExMSwicGF0aCI6Ii83Mzk3Njc5Ni80MDU1MTYzMzgtNDQzMjJlZDctZGY1ZC00MmRlLTg4MjUtOGU2NTAyYTUxNTExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTIyVDA3NDgzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI5ZjcwMTI3Yjk2NzY0MjQ3OWIxMTA2OTA1NmUyOGViYWE0MTFjYzA2YmYzNDhjYzRmZjJkZmMwZGU5YzY1NWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jAENs8v5rP2C2mXVI-K2vyvTUSvzKbCLckFjVSjP5mI)

应该是网络问题，挂载了个域名，可以试试。https://designable.tslang.org/

代码对应的这个仓库 https://github.com/erweixin/designable   [ formily/antd](https://github.com/erweixin/designable/blob/next/formily/antd/package.json)  这个路径下的项目。

**Reply by nan-184** [reply]
感谢大佬指点；Thanks♪(･ω･)ﾉ

**Reply by nan-184** [reply]
<img width="1428" alt="image" src="https://github.com/user-attachments/assets/e9feac89-dc3e-4f4f-bf36-2de912203af8" />
大佬   有没有这个设计器的使用文档，设计器中的响应器规则没看出来怎么去配置

**Reply by nimo141** [reply]
请问一下 现在designable还是只能支持到17吗 18是不是用不了

**Reply by SX729** [reply]
是在这个路径下install  然后start?

**Reply by SX729** [reply]
怎么运行的 指导下

**Reply by nimo141** [reply]
> 是在这个路径下install 然后start?

<img width="630" height="302" alt="image" src="https://github.com/user-attachments/assets/41f3a753-2a88-4c33-88a2-885b9a7b469c" />
示例文件内的整个放项目里去 解决依赖兼容问题后能正常运行

**Reply by SX729** [reply]
这个designable文件夹能单独做一个项目吗？还是放到我的react项目src里随便一个位置？这里面不是有自己的packagejson么 要单独安装吗？

**Reply by nimo141** [reply]
> 这个designable文件夹能单独做一个项目吗？还是放到我的react项目src里随便一个位置？这里面不是有自己的packagejson么 要单独安装吗？

<img width="603" height="638" alt="image" src="https://github.com/user-attachments/assets/cc23bb86-894b-479b-9ad5-010d08971223" />

<img width="1528" height="596" alt="image" src="https://github.com/user-attachments/assets/c17179f9-9c80-4799-a55b-c08df16580a6" />

可以参考我的路径 我放在组件文件夹中 系统中使用直接渲染main.tsx就行了

**Reply by SX729** [reply]
能给我你的个工程地址吗，我实在是起不来，，，，发愁

**Reply by SX729** [reply]
还有你的node 和 npm  都是什么版本 我在这试了好多版本了，老踩坑

**Reply by nimo141** [reply]
> 还有你的node 和 npm 都是什么版本 我在这试了好多版本了，老踩坑

https://gitee.com/nimo141/alibaba-form-formly-demo.git   这是我的demo地址  node是20.19.4 兼容问题找ai 我也是一边问ai一边改的

**Reply by SX729** [reply]
谢谢，请问你的表单下拉选项值来自接口的话，是怎么搞得，他会走原来项目的请求加header或者返回拦截等逻辑吗

**Reply by nimo141** [reply]
> 谢谢，请问你的表单下拉选项值来自接口的话，是怎么搞得，他会走原来项目的请求加header或者返回拦截等逻辑吗

Formily官网有示例 或者你可以用cursor跑一个demo 这个我们也是拿ai在改 没文档根本改不动

**Reply by SX729** [reply]
<img width="1682" height="682" alt="image" src="https://github.com/user-attachments/assets/e596619d-53e2-41bc-8930-dec740c3bc3b" />
我还是起不来。这个不是pnpm install 之后进来执行吗？

**Comment by aceHubert** [comment]
因为 react 19 不提供 umd 文件了，https://unpkg.com/react/umd/react.production.min.js 这个路由会 unpkg 会自动转到19版本上去，找不到文件
